db/migrate/20161007101725_create_orm_resources.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 class CreateOrmResources < ActiveRecord::Migration[5.0]
   def change
-    create_table :orm_resources, id: :uuid do |t|
-      t.jsonb :metadata, null: false, default: {}
-      t.timestamps
+    if ENV["VALKYRIE_ID_TYPE"] == "string"
```

Comment:
Can we make this a part of Valkyrie.config instead of having a special env variable we check?

Reply:
👍 that would be nice — then the test could change the config value and be able to test both UUIDs and strings.

Reply:
No. This should not go in config.  It's only used for changing the migration. it should probably (?) only be used for testing valkyrie.

Reply:
Probably true. An application could set this env when they perform the migrations, and that would be ok, but ultimately this is so that an application can do something like this https://github.com/jgondron/valkyrie_app/blob/master/db/migrate/20171214184147_change_id_to_string.rb at any time and we can confidently say that valkyrie will support it. I've fixed the issues I've found, but the only way to confidently say that going forward is if we can run specs against it when it's using string to prevent regression.